.github/ISSUE_TEMPLATE/typo-report.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Typo report
+about: Create a report to help us improve
+title: ''
+labels: typo
+assignees: ''
+
+---
+
+**Describe the bug**
```

Comment:
Is it strange to use the word "bug" here?

Reply:
```suggestion
**Describe the typo**
```